### PR TITLE
Fixed xgboost error

### PR DESCRIPTION
**Pull request by SiddhantSadangi**
# Description

Fixed xgboost example test error

__Related to:__ `fix test errors`

__Any expected test failures?__

---

Add a `[X]` to relevant checklist items

## ❔ This change

- [ ] adds a new feature
- [X] fixes breaking code
- [ ] is cosmetic (refactoring/reformatting)

---

## ✔️ Pre-merge checklist

- [ ] Refactored code ([sourcery](https://sourcery.ai/))
- [X] Tested code locally
- [X] Precommit installed and run before pushing changes
- [ ] Added code to GitHub tests ([notebooks](workflows/test-notebooks.yml), [scripts](workflows/test-scripts.yml))
- [ ] Updated GitHub [README](../README.md)
- [ ] Updated the projects overview page on Notion

---

## 🧪 Test Configuration

- OS: Windows11
- Python version: 3.8.15
- Neptune version: 0.16.17
- Affected libraries with version:

### CODE REVIEW

File: integrations-and-supported-tools/xgboost/scripts/Neptune_XGBoost_sklearn_api.py [hunk 1]
```diff
@@ -38,7 +38,6 @@
     X_train,
     y_train,
     early_stopping_rounds=30,
-    eval_metric=["mae", "rmse"],
```
Comment (kshitij12345) [inline integrations-and-supported-tools/xgboost/scripts/Neptune_XGBoost_sklearn_api.py:41]: What was the issue here?

Comment (Blaizzy) [reply integrations-and-supported-tools/xgboost/scripts/Neptune_XGBoost_sklearn_api.py:41]: I'm curious as well

Comment (SiddhantSadangi) [reply integrations-and-supported-tools/xgboost/scripts/Neptune_XGBoost_sklearn_api.py:41]: `eval_metrics` were being passed twice.. first during initialization (L34), and then again during `fit()` here.
This led to the 

> ValueError: 2 different `eval_metric` are provided.

**Review by kshitij12345 — APPROVED** [review]
LGTM

**Review by Blaizzy — APPROVED** [review]
LGTM

**Pull request merged**